java/javabuild.json

```diff
@@ -0,0 +1,112 @@
+{
+   "kind": "Template",
+   "apiVersion": "v1",
+   "metadata": {
+      "name": "oshinko-java-spark-build",
+      "annotations": {
+         "description": "Create a buildconfig and imagestream using STI and java spark source hosted in git"
+      }
+   },
+   "labels": {
+      "application": "oshinko-java-spark",
+      "createdBy": "template-oshinko-java-spark-build"
+   },
+   "parameters": [
+      {
+         "description": "The name to to use for the buildconfig and imagestream",
+         "name": "APPLICATION_NAME",
+         "generate": "expression",
+         "from": "java-spark-[a-z0-9]{4}",
+         "required": true
+      },
+      {
+         "description": "Name of the main py file to run",
```

Comment:
Still thinking about what to do with this.  With the *.jar search stuff, this is not necessary.  Unless for some reason your pom produces more than one jar ... which we could just say is not allowed.